tests/tls.go

```diff
@@ -66,7 +68,9 @@ func (s *TLSFunctionalSuite) SetupTest() {
 }
 
 func (s *TLSFunctionalSuite) TearDownTest() {
-	s.sdkClient.Close()
+	if s.sdkClient != nil {
```

Comment:
does TearDownTest get called if SetupTest does Skip()? I wouldn't have expected that

Reply:
It's called. The tests were panicking before this change.